Workiva/wf-grunt
# Typescript compilation and linting

### Add TypeScript Compilation Support

This adds compilation for TypeScript. 
### How to Verify

```
git fetch && git checkout ts
cd test/samples
grunt
```
- No build errors
- It should have compiled Animals.ts to dist/Animals.js
- It should also have Foo.js in the dist folder

@timmccall-wf
